container-images/scripts/build_llama_and_whisper.sh

```diff
@@ -14,16 +15,21 @@ dnf_install() {
     dnf install -y "$url"
     crb enable # this is in epel-release, can only install epel-release via url
     dnf --enablerepo=ubi-9-appstream-rpms install -y "${rpm_list[@]}"
-    local uname_m
-    uname_m="$(uname -m)"
-    dnf copr enable -y slp/mesa-krunkit "epel-9-$uname_m"
-    url="https://mirror.stream.centos.org/9-stream/AppStream/$uname_m/os/"
-    dnf config-manager --add-repo "$url"
-    url="http://mirror.centos.org/centos/RPM-GPG-KEY-CentOS-Official"
-    curl --retry 8 --retry-all-errors -o \
-      /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official "$url"
-    rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official
-    dnf install -y mesa-vulkan-drivers "${vulkan_rpms[@]}"
+    # x86_64 and aarch64 means kompute
+    if [ "$(uname -m)" = "x86_64" ] || [ "$(uname -m)" = "aarch64" ] ;then
+      local uname_m
+      uname_m="$(uname -m)"
```

Comment:
Just do:

local uname_m
uname_m="$(uname -m)"
    
before this if block and then use the $uname_m variable after this point everywhere.